### PR TITLE
build_deploy

### PR DESCRIPTION
**Pull request by ghost**
initial push for the build_deploy script that will be used to automate UHC CLI release process.
Currently only builds the binary and calculates the SHA sum.

### CODE REVIEW

**Comment by ghost** [comment]
@jhernand please take a look

**Comment by ALimaRedHat** [comment]
[test]

File: build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2018 Red Hat, Inc.
```
Comment (jhernand) [inline build_deploy.sh]: Should be 2019 :-) .

File: build_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script is executed by a Jenkins job for each change request. If it
+# doesn't succeed the change won't be merged.
+
+# make and rename uhc binary
```
Comment (jhernand) [inline build_deploy.sh]: We are in the process of renaming `uhc` to `ocm` (for _OpenShift Cluster Manager_) so the less you use the ` uhc` word the better. So don't use it in comments, if possible.

File: build_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+make
+mv uhc uhc-linux-amd64
+
+# calculate SHA 256 hash of the uhc binary
+sha256sum uhc-linux-amd64 > uhc-linux-amd64.sha256
```
Comment (jhernand) [inline build_deploy.sh]: GitHub is complaining that there is no line end character here.

**Review by jhernand — COMMENTED** [review]
Looks good to me. With some very minor comments.

File: build_deploy.sh (hunk 2 above)
Comment (ghost) [reply build_deploy.sh]: I was going to ask about this actually, good to know its already a WIP.

**Pull request merged**